samples/bluetooth/mesh/dfu/target/src/main.c

```diff
@@ -14,6 +14,9 @@
 #include <zephyr/bluetooth/bluetooth.h>
 #include <zephyr/bluetooth/mesh.h>
 #include "dfu_target.h"
+#if IS_ENABLED(CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU)
```

Comment:
```suggestion
#if defined(CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU)
```
https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/sys/util_macro.h#L117-L118